app/src/main/admin/components/ModellingGroups/SingleGroup/Details/ModellingGroupDetailsContent.tsx

```diff
@@ -8,17 +10,16 @@ import { GroupMembersSummary } from "./GroupMembersSummary";
 import { userStore } from "../../../../stores/UserStore";
 
 import "../../../../../shared/styles/common.scss";
-import {adminAuthStore} from "../../../../stores/AdminAuthStore";
 
 interface Props extends RemoteContent {
     group: ModellingGroupDetails;
     users: User[];
-    canManageGroupMembers: boolean;
+    canManageGroupMembers?: boolean;
```

Comment:
Again I think a nicer way is to leave this non-null but make the method below return `Partial<Props>`